cluster-scope/base/groups/cluster-admins/group.yaml

```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: user.openshift.io/v1
+kind: Group
+metadata:
+  name: cluster-admins
```

Comment:
should we change this to `opf-cluster-admins` to avoid any possible future conflicts?

Reply:
I think this is generic. This group comes from MOC implementation actually and clones the live resource. It also includes the cluster admins from BU for the MOC cluster. I agree with the possible conflict, but we aim to have the whole cluster ArgoCD managed - therefore we should not be afraid to touch these groups as well. :slightly_smiling_face: